cmd/tpm-manager/tpm-manager.go

```diff
@@ -0,0 +1,506 @@
+	fs.IntVar(&manager.recurring, "reverify", 0, "Periodocally reverify nodes after this many seconds")
+}
+
+func updateConfig(configmap *api.ConfigMap) {
+	if configmap.Data["allowunknown"] != "" {
+		allowunknown, err := strconv.ParseBool(configmap.Data["allowunknown"])
+		if err == nil {
+			manager.allowunknown = allowunknown
+		}
+	}
+	if configmap.Data["reverify"] != "" {
+		reverify, err := strconv.Atoi(configmap.Data["reverify"])
+		if err == nil {
+			manager.recurring = reverify
+			select {
+			case manager.recurringChan <- reverify:
+			default:
+			}
+		}
+	}
+}
+
+func run(stop <-chan struct{}) {
+	_, nodeController := framework.NewInformer(
+		&cache.ListWatch{
+			ListFunc: func(options api.ListOptions) (runtime.Object, error) {
+				return manager.client.Nodes().List(options)
+			},
+			WatchFunc: func(options api.ListOptions) (watch.Interface, error) {
+				return manager.client.Nodes().Watch(options)
+			},
+		},
+		&api.Node{},
+		controller.NoResyncPeriodFunc(),
+		framework.ResourceEventHandlerFuncs{
+			AddFunc:    nodeAddFn,
+			UpdateFunc: nodeUpdateFn,
+			DeleteFunc: nodeDeleteFn,
+		},
+	)
+
+	_, configController := framework.NewInformer(
+		&cache.ListWatch{
+			ListFunc: func(options api.ListOptions) (runtime.Object, error) {
+				return manager.client.ConfigMaps("default").List(options)
+			},
+			WatchFunc: func(options api.ListOptions) (watch.Interface, error) {
+				return manager.client.ConfigMaps("default").Watch(options)
+			},
+		},
+		&api.ConfigMap{},
+		controller.NoResyncPeriodFunc(),
+		framework.ResourceEventHandlerFuncs{
+			AddFunc:    configAddFn,
+			UpdateFunc: configUpdateFn,
+			DeleteFunc: configDeleteFn,
+		},
+	)
+
+	_, policyController := framework.NewInformer(
+		&cache.ListWatch{
+			ListFunc: func(options api.ListOptions) (runtime.Object, error) {
+				return manager.tpmhandler.PolicyClient.List(&options)
+			},
+			WatchFunc: func(options api.ListOptions) (watch.Interface, error) {
+				return manager.tpmhandler.PolicyClient.Watch(&options)
+			},
+		},
+		&runtime.Unstructured{},
+		controller.NoResyncPeriodFunc(),
+		framework.ResourceEventHandlerFuncs{
+			AddFunc:    policyAddFn,
+			UpdateFunc: policyUpdateFn,
+			DeleteFunc: policyDeleteFn,
+		},
+	)
+	fmt.Printf("Starting\n")
+	getPolicy()
+	go reverify()
+	go nodeController.Run(wait.NeverStop)
+	go configController.Run(wait.NeverStop)
+	go policyController.Run(wait.NeverStop)
+	select {}
+	fmt.Printf("Returned\n")
+}
+
+func main() {
+	var tpmhandler tpm.TPMHandler
+	var config restclient.Config
+
+	//	config, err := clientcmd.BuildConfigFromFlags(manager.Master, manager.Kubeconfig)
+	err := client.SetKubernetesDefaults(&config)
+	if err != nil {
+		fmt.Errorf("Unable to create client configuration: %v", err)
+	}
+	config.Host = "http://localhost:8080"
+	client, err := client.New(&config)
+	if err != nil {
+		fmt.Printf("Unable to create client: %v", err)
+		return
+	}
+	tpmhandler.Setup(&config)
```

Comment:
nit: style convention would be 

```
tpmHandler := NewTPMHandler(&config)
```

rather than 

```
var tpmhandler tpm.TPMHandler
tpmhandler.Setup(&Config)
```